chain/store/store.go

```diff
@@ -47,6 +47,7 @@ var (
 
 var DefaultTipSetCacheSize = 8192
 var DefaultMsgMetaCacheSize = 2048
+var DefaultTipSetMessageCacheSize = 8192
```

Comment:
This seems too aggressive. How did we arrive to this number? Assuming an average of 150 messages per tipset, this could be holding a total of 1228800 messages. And if we store this, do we even need the above message meta cache any longer?

Reply:
I just put some number there when I was working on the PR, it was still in draft. I am not sure what number to put here, how large are each message on average? Anyway, I lowered it to 1024 but that may still be too high?